cpp/ql/lib/experimental/semmle/code/cpp/semantic/analysis/RangeAnalysisStage.qll

```diff
@@ -0,0 +1,1011 @@
+/**
```

Comment:
Do I see correctly that this is just a parameterized copy of what was in `RangeAnalysis.qll` before, or are there any differences I'm overlooking?

Reply:
For now it's just a parameterized copy. I'm working on a followup PR that will add some overflow predicates to the `Lang` parameter and start using the parameterized module to generate two separate stages.